src/xdocs/config_imports.xml

```diff
@@ -818,6 +822,42 @@ import android.*;
           which is part of the Checkstyle distribution.
         </p>
 
+        <h4 id="blacklist-example">Example of blacklist mode</h4>
+        <p>
+          Short description of the behaviour:
+          <ul>
+            <li>Check starts checking from the closest to the current package described inside import control file.</li>
+            <li>If there is matching allow/disallow rule inside the current package then check returns "allowed" or "disallowed" message.</li>
```

Comment:
"current package" ==> "current subpackage" . As in import files all are sunpackages
"then check" ==> "then the Check"